locations/spiders/vets4pets_gb.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+from scrapy.spiders import CrawlSpider, Rule
+from scrapy.linkextractors import LinkExtractor
+from locations.linked_data_parser import LinkedDataParser
+from locations.spiders.petsathome_gb import PetsAtHomeGBSpider
+from locations.google_url import url_to_coords
+
+
+def extract_google_position(item, response):
+    for link in response.xpath("//iframe/@src").extract():
+        if "google.com" in link:
+            item["lat"], item["lon"] = url_to_coords(link)
+            return
+
+
+def set_located_in(item, location):
+    item["located_in"] = location["brand"]
+    item["located_in_wikidata"] = location["brand_wikidata"]
+
+
+class Vets4PetsGBSpider(CrawlSpider):
+    name = "vets4pets_gb"
+    item_attributes = {
+        "brand": "Vets4Pets",
+        "brand_wikidata": "Q16960196",
+    }
+    start_urls = ["https://www.vets4pets.com/practices/"]
+    rules = [
+        Rule(LinkExtractor(allow="/practices/"), callback="parse_func", follow=True)
+    ]
+    allowed_domains = ["vets4pets.com"]
+    download_delay = 0.2
+
+    def parse_func(self, response):
+        ld = LinkedDataParser.find_linked_data(response, "VeterinaryCare")
+        if ld:
+            # Should we support lists of addresses in the common parser code and take the first?
+            ld["address"] = ld["address"][0]
```

Comment:
I don't know if it's [to spec], but it seems reasonable to use the first in an array. I'd vote for changing the parser.

Reply:
https://schema.org/Organization

Reply:
I will do this.